calculator.rb

```diff
@@ -0,0 +1,75 @@
+def add(num_1, num_2)
+  return  "\n#{num_1} + #{num_2} = #{num_1 + num_2} "
+end
+def substract(num_1, num_2)
+  return "\n#{num_1} - #{num_2} = #{num_1 - num_2} "
+end
+def multiply(num_1, num_2)
+  return "\n#{num_1} * #{num_2} = #{num_1 * num_2} "
+end
+def divide(num_1, num_2)
+  return "\n#{num_1} / #{num_2} = #{num_1 / num_2} "
+end
+
+operator_array = ["add","+", "substract","-", "multiply","*", "divide","/"]
+puts  "\nWelcome to the Calculator Program!"
+print "How many operations would you like to do?: " 
+operator_num = gets.chomp
+
+while operator_num !~ /^-?[0-9]+$/
+  puts "NUMBERS ONLY!!!!!"
+  print "\nHow many operations would you like to do?: " 
+  operator_num = gets.chomp
+end 
+operator_num = operator_num.to_i
+
+operator_num.times do |i|
+  puts "\n\nWhich operator would you like to use?"
+  puts "\n1. add (+)\n2. substract (-)\n3. multiply (*)\n4. divide (/)"
+  print "\n##{i + 1} Please choose one operator(name or symbol): "
+  operator = gets.chomp.downcase
+
+
+  while  !operator_array.include?(operator)
+    puts "Please enter valid operator:"
+    print "\n##{i + 1} Please chooce one operator(name or symbol): "
+    operator = gets.chomp
+  end
+  operator = operator
+  if operator_array.include?(operator)
```

Comment:
You've written almost exactly the same code here twice, to get the first number and the second number. Could you DRY that up by putting this logic in a method?